ntbs-build/openshift-pipelines/tasks/download-file-from-container.yml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
This file represents an earlier (slightly offbeat) solution to the problem that I didn't end up going with. In this approach, the dacpac was wrapped in a docker container which was stored in a github registry. There were advantages, and disadvantages to this approach, but the killer blow was that this Task was not able to run in parallel with itself.

Reply:
I've left the Task in OpenShift (and therefore included it in this commit) in case we want to revisit it in future. However, on reflection I think it may have been better to go the other way and delete it from both instead.